drivers/mag3110/mag3110_saul.c

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (C) 2017 HAW Hamburg
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     driver_mag3110
+ * @{
+ *
+ * @file
+ * @brief       MAG3110 adaption to the RIOT actuator/sensor interface
```

Comment:
both are correct, I leave as is 'cause its written that way in every other saul adaption